beets/ui/commands.py

```diff
@@ -1479,17 +1479,20 @@ def move_items(lib, dest, query, copy, album, pretend, confirm=False):
             show_path_changes([(obj.path, obj.destination(basedir=dest))
                                for obj in objs])
     else:
-        if confirm:
-            objs = ui.input_select_objects(
-                u'Really %s' % act, objs,
-                lambda o: show_path_changes(
-                    [(o.path, o.destination(basedir=dest))]))
+        if export:
+            util.copy(item.path, item.destination(basedir=dest))
```

Comment:
File "/home/travis/build/beetbox/beets/test/test_ui.py", line 446, in test_move_album
    self.assertNotExists(self.itempath)
  File "/home/travis/build/beetbox/beets/test/_common.py", line 140, in assertNotExists
    u'file exists: {!r}'.format((path)))
AssertionError: True is not false : file exists: b'/tmp/tmpqr4bm0wx/testlibdir/srcfile'

This test asserted that the album didn't existed but it turned out that it existed?